src/porepy/numerics/darcyEq.py

```diff
@@ -0,0 +1,175 @@
+import numpy as np
+import scipy.sparse as sps
+
+from porepy.numerics.fv import tpfa, source
+from porepy.grids.grid_bucket import GridBucket
+from porepy.params import bc, tensor
+from porepy.params.data import Parameters
+from porepy.viz.exporter import export_vtk
+
+class Darcy():
+    def __init__(self, gb, data=None, physics='flow'):
+        self.physics = physics
+        self._gb = gb
+        self._data = data
+        self.lhs, self.rhs = self.reassemble()
+        self.p = np.zeros(self.flux_disc().ndof(self.grid()))
+        self.parameters = {'file_name': physics}
+        self.parameters['folder_name'] = 'results'
+
+    def solve(self):
+        self.p = sps.linalg.spsolve(self.lhs, self.rhs)
+        return self.p
+
+    def reassemble(self):
+        """
+        reassemble matrices. This must be called between every time step to
```

Comment:
Right hand side, yes, but there is no need to reassemble the discretization between time steps?

Reply:
I can think of cases where lhs reassembly is necessary. One example is if you want to use the same setup with different parameters, say change permeability in one fracture to study impact of connectivity

Reply:
Could add something like: 
def reasemblerhs(self):
or 
def reasemble(self, only_rhs=False)